### DISCUSSION TITLE
Curve at specific temp, auto when lower

### DISCUSSION THREAD

**Discussion by rbcetin** [Rem0o/FanControl.Releases#1413] [Ideas]
Hello guys, with Zotac 4090 AMP, you can't set fan speed under %30. But vbios has fan-stop mode. Under 47 celcius it will stop the fans. But you can not do that with custom fan curve, not even with Afterburner. 

So if we can have something like,

Trigger Mode:
50 celcius -> Custom curve
45 celcius -> Auto mode

With this way we can use the fan-stop on vbios at some temp. 

So for card that does not support 0rpm with custom fan curves, this will be a lifesaver. At some specified temp custom curve is disabled, at some temp its enabled.

Right now this is trying to set %0 for the fan speed, since card does not permit this, fans are running at 30%.

![image](https://user-images.githubusercontent.com/70274474/209466918-561b2ee6-5917-4e94-8d5a-cf226ad768da.png)

If i disable the curve, then fans will stop.

![image](https://user-images.githubusercontent.com/70274474/209466927-4c1ac0ad-0922-4814-ad95-5eda3261e370.png)

So basically, at some temperature I want to disable the curve automatically, at some temp I want to enable it again.

A curve with two different phases at some temp its AUTO after some temp a custom curve.

**Comment by Rem0o** [comment]
![image](https://user-images.githubusercontent.com/14118956/209483772-b73c9c61-570d-4752-9511-bba4e880fdd0.png)

Read the "?" carefully.
